main.go

```diff
@@ -147,3 +164,12 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 		}
 	}
 }
+
+func initRedisClient() {
```

Comment:
Just in-line this function in the init, the go-compiler is just going to in-line it for you so might as well do it in the actual code base